update_sc_name/update_scientific_name.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import os.path
+from argparse import ArgumentParser
+
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import execute_query
+from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_eva, normalise_taxon_scientific_name
+
+
+def update_path_reference_sequence(taxonomy, old_sc_name, ref_seq_dir, new_sc_name):
+    species_ref_dir = os.path.join(ref_seq_dir, normalise_taxon_scientific_name(old_sc_name))
+    if not os.path.exists(species_ref_dir):
+        logging.warning(f'For taxonomy {taxonomy}, could not find any directory with name '
+                        f'"{normalise_taxon_scientific_name(old_sc_name)}" in reference sequence directory {ref_seq_dir}')
+    else:
+        new_species_ref_dir = os.path.join(ref_seq_dir, normalise_taxon_scientific_name(new_sc_name))
+        os.rename(species_ref_dir, new_species_ref_dir)
```

Comment:
I would prefer creating a symlink rather than renaming. 
Also we should check that the target does not exist already as we might need to merge two directories instead of creating a new one.

Reply:
updated